js/views/root/Content.jsx

```diff
@@ -12,22 +12,13 @@ class RootContent extends React.Component {
   }
 
   state = {
-    stations: StationStore.getData(),
+    stations: null,
```

Comment:
i also changed this to do an initial update on CDM

Reply:
okay, as long as it renders I guess 🤷‍♀